### DISCUSSION TITLE
Differences between GEOM and OBST

### DISCUSSION THREAD

**Discussion by lihy213** [firemodels/fds#13077] [Q&A]
Dear developers,

In recent versions (e.g., FDS 6.9.1), the `GEOM` keyword was introduced to increase the applicability of FDS to unstructured geometry. It allows the division of an object's surface into multiple triangular meshes using various definitions, primarily involving blocks, spheres, cylinders, and so on.  But the current `GEOM` function also has some limitations, particularly in relation to `VENT`, `MULT`, and similar functions.It's worth noting that the more commonly used OBST function can also create geometry, but it uses structural cuts. My question is, what are the differences between `GEOM` and `OBST` in terms of application, such as geometry modeling and fire simulation?  While I am aware that both can configure faces to undergo burning in some manner, I'm uncertain whether, in most cases, the geometry created by `GEOM` should be considered as an entity that is not involved in any computations during use?

Regards,

LI

**Comment by rmcdermo** [comment]
FDS user guide, 7.3.11 GEOM Limitations
